sceptre/strides/config/prod/aws-cconfig.yaml

```diff
@@ -0,0 +1,12 @@
+template_path: "aws-config.yaml"
+stack_name: "aws-config"
+stack_tags:
+  Department: "Platform"
+  Project: "Infrastructure"
+  OwnerEmail: "it@sagebase.org"
+dependencies:
+  - "prod/cloudtrail.yaml"
+parameters:
+  OperatorEmail: "aws.strides@sagebase.org"
+  LogCentralBucket: "essentials-awss3configbucket-9n8wjykhvr5z"  # bucket in org-sagebase-logcentral account
```

Comment:
This is the name of the old logging bucket, right?

Reply:
good catch, i'll need to update this to new bucket after merging PR #122

Reply:
updated to new bucket